### ISSUE TITLE
./vendor/bin/toothpaste: No such file or directory

### ISSUE THREAD

**Issue by R3dian** [esimonetti/toothpaste#2]
Hi Enrico,

I am trying to set it up on a ubuntu machine but getting the following error after installing everything via the composer.

> ./vendor/bin/toothpaste: No such file or directory

Under the "vendor/bin" directory I can only see "phpunit" file but no toothpaste files in there.

Any idea what could be the reason?

Kind Regards,
Junaid

**Comment by esimonetti** [comment]
Hi Junaid,

That's odd, would you be able to provide a little more info?

Let me share with you what I use and the output I get, and see if that helps

```
sugar@a192867c88d4:/var/www/html/toothpastetmp$ composer --version
Composer version 1.9.3 2020-02-04 12:58:49

sugar@a192867c88d4:/var/www/html/toothpastetmp$ php -v
PHP 7.3.13 (cli) (built: Dec 18 2019 23:09:33) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.13, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.13, Copyright (c) 1999-2018, by Zend Technologies

sugar@a192867c88d4:/var/www/html/toothpastetmp$ composer require esimonetti/toothpaste dev-master
./composer.json has been created
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 11 installs, 0 updates, 0 removals
  - Installing ralouphie/getallheaders (3.0.3): Downloading (100%)         
  - Installing psr/http-message (1.0.1): Downloading (100%)         
  - Installing guzzlehttp/psr7 (1.6.1): Downloading (100%)         
  - Installing guzzlehttp/promises (v1.3.1): Downloading (100%)         
  - Installing guzzlehttp/guzzle (6.4.1): Downloading (100%)         
  - Installing psr/container (1.0.0): Downloading (100%)         
  - Installing symfony/service-contracts (v1.1.9): Downloading (100%)         
  - Installing symfony/polyfill-php73 (v1.18.0): Downloading (100%)         
  - Installing symfony/polyfill-mbstring (v1.18.0): Downloading (100%)         
  - Installing symfony/console (v4.3.11): Downloading (100%)         
  - Installing esimonetti/toothpaste (dev-master e6b958f): Cloning e6b958fdfa from cache
guzzlehttp/psr7 suggests installing zendframework/zend-httphandlerrunner (Emit PSR-7 responses)
guzzlehttp/guzzle suggests installing psr/log (Required for using the Log middleware)
symfony/service-contracts suggests installing symfony/service-implementation
symfony/console suggests installing symfony/event-dispatcher
symfony/console suggests installing symfony/lock
symfony/console suggests installing symfony/process
symfony/console suggests installing psr/log (For using the console logger)
Writing lock file
Generating autoload files

sugar@a192867c88d4:/var/www/html/toothpastetmp$ cat composer.json 
{
    "require": {
        "esimonetti/toothpaste": "dev-master"
    }
}

sugar@a192867c88d4:/var/www/html/toothpastetmp$ ./vendor/bin/toothpaste --version
Toothpaste v0.2.1
Toothpaste 0.2.1

sugar@a192867c88d4:/var/www/html/toothpastetmp$ ls -la vendor/esimonetti/toothpaste/bin/          
total 16
drwxr-xr-x 2 sugar sugar 4096 Jul 16 09:54 .
drwxr-xr-x 7 sugar sugar 4096 Jul 16 09:54 ..
-rwxr-xr-x 1 sugar sugar  117 Jul 16 09:54 toothpaste
-rw-r--r-- 1 sugar sugar 2090 Jul 16 09:54 toothpaste.php
```

**Comment by R3dian** [comment]
Hi Enrico,

The following are the steps I performed.

- Fork the repository.
- Ran git clone on my local machine.
- Now in the clone directory, I ran "composer require esimonetti/toothpaste dev-master". The only difference in the above output and mine is the following line missing for me.
"Installing esimonetti/toothpaste (dev-master e6b958f): Cloning e6b958fdfa from cache" 

**P.S.** I just ran the same command (empty directory) without taking the git clone and it worked.

Kind Regards,
Junaid
